spec/models/text_widget_spec.rb

```diff
@@ -35,10 +35,12 @@
       expect(widget.text_body_html).to eq "123"
     end
 
-    it "should be invalid with a non-spec'd key" do
-      widget.content['not_a_real_field'] = "heyy"
-      expect(widget).not_to be_valid
-    end
+    ## Not sure this is necessary. Even if someone
+    # should access +content+ directly, the object
+    # won't have a getter, so the value would be hidden
+    #   widget.content["blah"] = "carrots"
+    #   widget.blah // raises undefined method
+    it "should be invalid with a non-spec'd key"
```

Comment:
OK. We can block direct access to `#content` (another PR). But really it's the job of the controller to filter what params get through.